docs/personalization/integrate_recommendation_service.md

```diff
@@ -151,6 +151,63 @@ if ($recommendations && isset($recommendations->recommendationResponseList)) {
 curl_close($curl);
 ```
 
+## Send emails with recommendations
+
+By using [email triggers]([[= user_doc =]]/personalization/triggers.md), your organization can send emails to individual visitors, for example, to invite them to return to the website or remind them of items abandoned in a cart.
+
+Email triggers are push messages with recommendations.
+Specific conditions, such as the time that must pass before email start being sent, content types and attributes to be included in a response, or a number of repetitions, are first defined based on an interview between you and Ibexa.
+Email triggers are then processed on the Personalization server and responses are delivered to a dedicated endpoint.
+
+To let your visitors receive emails with recommendations:
+
+1\. With the [User API](api_reference/user_api.md#post-requests), add the `e-mail` attribute to the user record.
+
+2\. Prepare an endpoint that will intercept push messages and pass them on to your mailing system, for example, an Ibexa Connect [webhook](https://doc.ibexa.co/projects/connect/en/latest/tools/webhooks/). The webhook must meet the following requirements:
+
+   - cannot require authentication
+   - must support POST requests
+   - must accept JSON objects in a format that resembles the following example:
```

Comment:
Please update JSON example to match prepared template (we decided to simplify reco response): https://issues.ibexa.co/secure/attachment/41005/trigger_message_example.json